app/controllers/api/v1/auth/auth_controller.rb

```diff
@@ -0,0 +1,97 @@
+class Api::V1::Auth::AuthController < Api::V1::BaseController
+  before_action :set_user!, except: %i[extend_token sign_in_via_email_code]
```

Comment:
we don't need sign in via email code here in this codebase. its just login with email and password. can remove all code relating to that.